ffyaml/ffyaml_test.go

```diff
@@ -1,12 +1,12 @@
 package ffyaml_test
 
 import (
-	"testing"
-	"time"
-
+	"errors"
 	"github.com/peterbourgon/ff/v3"
 	"github.com/peterbourgon/ff/v3/fftest"
 	"github.com/peterbourgon/ff/v3/ffyaml"
+	"testing"
+	"time"
```

Comment:
Please update your editor to run `goimports` on save (or whatever). Import blocks should have two sections: stdlib, newline, non-stdlib.

Reply:
Sorry. I have added in an auto-run `goimports` entry now upon save.